test/unit_test/rag/nlp/test_default_delimiter_constant.py

```
@@ -0,0 +1,232 @@
+# ---------------------------------------------------------------------------
+# Constant
+# ---------------------------------------------------------------------------
+
+
+def test_default_delimiter_constant_value():
+    """The constant must be the full set: newline + ``!?;`` (English
+    punctuation) + ``。，！？`` (Chinese punctuation). The ``;`` is
+    the one the pre-fix defaults in docx / image / email dropped --
+    see issue #18562.
+    """
+    assert DEFAULT_DELIMITER == "\n!?;。；！？", (
```

Ruff (0.16.1)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)